addons/website_slides/tests/test_slide_question.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.website_slides.tests import common as slides_common
+from odoo.tests import tagged
+from odoo.tests.common import users
+
+@tagged('slide_question')
+class TestSlideQuestionManagement(slides_common.SlidesCase):
+
+    @users('user_officer')
+    def test_compute_answers_validation_error(self):
+        channel = self.env['slide.channel'].create({
+            'name': 'Test compute answers channel'
+        })
+
+        slide = self.env['slide.slide'].create({
+            'name': "Test compute answers validation error slide",
+            'channel_id': channel.id,
+            'slide_category': 'quiz',
+        })
+
+        question = self.env['slide.question'].create({
+            'question': 'Will test compute answers validation error pass?',
+            'slide_id': slide.id,
+        })
+
+        # incorrect_answer
+        self.env['slide.answer'].create({
+            'question_id': question.id,
+            'text_value': 'An incorrect answer',
+        })
```

Comment:
This can probably be done in fewer `create` calls, as you can specify `Command.create(...)` in `question_ids` for the channel, and probably do further nesting.
It could require making variables for the data instead of a giant single variable, but in the end we'd call create only once.